Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard_description.md

```diff
@@ -24,4 +24,4 @@ See the [Getting Started section](https://securityscorecard.readme.io/docs/getti
 
 ## Update Interval
 
-SecurityScorecard scores are updated on a daily basis. Therefore it's recommended to leave the Incidents Fetch Interval configuration to 1 day.
```

Comment:
We can avoid this change.

Reply:
I think the `format` chose + over - in the markdown